File: Count the Elements Greater Than the Given Number/count_elements_greater_than_given_number.c
#include <stdio.h>
int main() {
    int n;
    printf("Enter the number of elements: ");
    scanf("%d", &n);
    int a[n];
    for (int i = 0; i < n; i++) {
        printf("Enter element %d: ", i + 1);
        scanf("%d", &a[i]);
    }
    int x;
    printf("Enter the number to compare: ");
    scanf("%d", &x);
    int count = 0;
    for (int i = 0; i < n; i++) {
        if (a[i] > x) {
            count++;
        }
    }
    printf("Count of elements greater than %d is: %d\n", x, count);
    return 0;
}

File: Count the Elements Greater Than the Given Number/CountElementsGreaterThanGivenNumber.java
public class CountElementsGreaterThanGivenNumber {
    public static void main(String[] args) {
        java.util.Scanner sc = new java.util.Scanner(System.in);
        System.out.print("Enter the number of elements: ");
        int n = sc.nextInt();
        int[] a = new int[n];
        for (int i = 0; i < n; i++) {
            System.out.print("Enter element " + (i + 1) + ": ");
            a[i] = sc.nextInt();
        }
        System.out.print("Enter the number to compare: ");
        int x = sc.nextInt();
        int count = 0;
        for (int i = 0; i < n; i++) {
            if (a[i] > x) {
                count++;
            }
        }
        System.out.println("Count of elements greater than " + x + " is: " + count);
    }
}
